lua/tardis/languages/en.lua

```diff
@@ -93,6 +93,7 @@ T.Phrases = {
     ["Common.Save"] = "Save",
```

Comment:
Language files need to be updated to match these changes or they'll get wiped out

Reply:
Should be done